Generator/Sources/FileRenderer.swift

```diff
@@ -15,43 +15,109 @@
 protocol FileRenderer {
     var targetDirectory: String { get }
     var fileNamePrefix: String { get }
+    var context: Context { get }
+
     func renderFile(_ namespace: Namespace) throws -> String
+
+    /// Converts an attribute ID to a Swift member path based on the type structure of the renderer.
+    /// Example: "foo.bar_baz" -> "OTelAttribute.foo.barBaz"
+    func attributeIdToSwiftMemberPath(_ attributeId: String) throws -> String
 }
 
-func renderDocs(_ attribute: Attribute) -> String {
-    var result = "`\(attribute.id)`"
-    if let brief = attribute.brief?.trimmingCharacters(in: .whitespacesAndNewlines), !brief.isEmpty {
-        result.append(": \(brief)")
-    }
+/// Contains top-level informatino about the rendering execution
```

Comment:
```suggestion
/// Contains top-level information about the rendering execution
```

Reply:
Fixed in: https://github.com/swift-otel/swift-otel-semantic-conventions/pull/19/commits/4807300ea3b3a63190a981ad5dc45963d5c33b3e